### DISCUSSION TITLE
Disabling Dependabot Email Spam

### DISCUSSION THREAD

**Discussion by malekashkar** [community/community#36088] [Actions]
Hello, I've ran through every single guide on the internet on disabling email notifications from dependabot however nothing worked.
This is basically my final destination, I'm guessing someone has faced the same issue and knows how to fix this.

![image](https://user-images.githubusercontent.com/62209591/195743424-209089af-e01e-4a17-b3f7-5142475aefb2.png)

**Comment by jooy2** [comment]


**Reply by malekashkar** [reply]
In the future I'm planning on making some of my projects open source.

If this is going to cause an issue with PR's there then I would rather not, is there any other possible way? Unless it doesn't affect that.

**Comment by adbenitez** [comment]
I am having the same issue, I need to follow issues and pull requests and the dependabot notifications are a lot and polluting my inbox

if there is a way to block the dependabot user I am willing to do it
